flight/Modules/Actuator/actuator.c

```diff
@@ -61,6 +61,7 @@
 #define TASK_PRIORITY PIOS_THREAD_PRIO_HIGHEST
 #define FAILSAFE_TIMEOUT_MS 100
 #define MAX_MIX_ACTUATORS ACTUATORCOMMAND_CHANNEL_NUMELEM
+#define MULTIROTOR_MIXER_UPPER_BOUND 128
```

Comment:
Is this only used for multirotors? Using 

    #define MIXER_RANGE 127

would make it more consistent with the GCS code.